src/api.ts

```diff
@@ -24,4 +24,4 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser = (userId: number | undefined) => get<User>(`/users/${userId}`);
```

Comment:
Not fixed
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/54936044/2f27c6f8-74a5-425b-9a53-d74b6ec6b4a5)